service/src/main/java/bio/terra/lz/futureservice/common/utils/RequestQueryParamUtils.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.lz.futureservice.common.utils;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.UUID;
+import org.apache.commons.lang3.StringUtils;
+
+public class RequestQueryParamUtils {
+  private static final String BILLING_PROFILE_ID_PARAM_NAME = "billingProfileId";
+
+  private RequestQueryParamUtils() {}
+
+  public static boolean isBillingProfileIdSanitized(
```

Comment:
I wonder if we will need to generalize this method for other parameters… but that can wait I suppose.

Reply:
Agree. This can be useful. But I would postpone this until we need it.